src/zenml/integrations/smtp_email/alerters/smtp_email_alerter.py

```diff
@@ -0,0 +1,398 @@
+                    <p>This is an automated message from ZenML. Please do not reply to this email.</p>
+                  </div>
+                </div>
+              </body>
+            </html>
+            """
+            return html
+
+        # Simple HTML formatting if no payload
+        return f"""
+        <html>
+          <body style="font-family: Arial, sans-serif;">
+            <h2>ZenML Notification</h2>
+            <div>{formatted_message}</div>
+            <p><em>This is an automated message from ZenML.</em></p>
+          </body>
+        </html>
+        """
+
+    def post(
+        self,
+        message: Optional[str] = None,
+        params: Optional[BaseAlerterStepParameters] = None,
+    ) -> bool:
+        """Send an email alert.
+
+        Args:
+            message: Message content for the email.
+            params: Optional parameters.
+
+        Returns:
+            True if operation succeeded, else False
+        """
+        if message is None:
+            message = "ZenML Alert"
+
+        try:
+            recipient_email = self._get_recipient_email(params=params)
+            include_html = self._should_include_html(params=params)
+
+            # Create message
+            email_message = MIMEMultipart("alternative")
+            email_message["From"] = self.config.sender_email
+            email_message["To"] = recipient_email
+
+            # Determine subject
+            if (
+                params
+                and isinstance(params, SMTPEmailAlerterParameters)
+                and params.subject is not None
+            ):
+                subject = params.subject
+            else:
+                # Use default subject with prefix
+                subject = f"{self._get_subject_prefix()} {message[:50]}{'...' if len(message) > 50 else ''}"
+
+            email_message["Subject"] = subject
+
+            # Attach plain text version
+            email_message.attach(MIMEText(message, "plain"))
+
+            # Attach HTML version if enabled
+            if include_html:
+                html_body = self._create_html_body(message, params)
+                email_message.attach(MIMEText(html_body, "html"))
+
+            # Send email
+            server = smtplib.SMTP(
+                self.config.smtp_server, self.config.smtp_port
+            )
+            if self.config.use_tls:
+                server.starttls()
+            server.login(self.config.sender_email, self.config.password)
+            server.send_message(email_message)
+            server.quit()
+
+            logger.info("Email alert sent successfully")
+            return True
+
+        except Exception as e:
+            logger.error(f"Error sending email alert: {str(e)}")
+            return False
+
+    def ask(
+        self, question: str, params: Optional[BaseAlerterStepParameters] = None
+    ) -> bool:
```

Comment:
```suggestion
    ) -> Never:
```

`from typing import Never`